test/src/test/java/hudson/ProcStarterTest.java

```diff
@@ -45,6 +45,7 @@
 import hudson.tasks.BuildWrapperDescriptor;
 import hudson.tasks.Builder;
 
+
```

Comment:
Unnecessary whitespace change is no big deal here, but it's generally better to minimize occurrences of something like this before you submit a PR.